app/interfaces/api/v2/mi_query_helper.rb

```diff
@@ -0,0 +1,40 @@
+module API::V2
+  module MIQueryHelper
+    extend Grape::API::Helpers
```

Comment:
I know this is currently being used for the **API** only, but the class itself seems to be defining a query only. I would suggest this is in a called something like `Claims::MI::Queries` (as a module/class) inside `app/models/claims/mi/queries' and you could just call it as:

```ruby
Claims::MI::Queries.scheme_ten_claims(start_date, end_date)
```

Doesn't feel right having all that `SQL` as a Grape helper.